### ISSUE TITLE
The link provided in the about section isn't working: https://booksuno.xyz/

### ISSUE THREAD

**Issue by vignesh1507** [anonthedev/booksuno#9]
This link isn't working: https://booksuno.xyz/

**Comment by anonthedev** [comment]
works fine for me, maybe send a screenshot of what's it showing?

**Comment by vignesh1507** [comment]
<img width="1440" alt="Screenshot 2024-10-10 at 2 00 00 PM" src="https://github.com/user-attachments/assets/31e4d6ff-0c36-40c8-b1fd-5a123a7f9719">

**Comment by anonthedev** [comment]
u didn't upload the screen shot

**Comment by vignesh1507** [comment]
Sorry, my bad...uploaded the screenshot again you can check that out.

**Comment by anonthedev** [comment]
on what wifi are you trying to access it?

**Comment by vignesh1507** [comment]
i tried using STUD the website didn't open but when i try to open it using my mobile hotspot then the website is opening but the audio isn't working in the website.

**Comment by anonthedev** [comment]
yeah STUD has it banned ig, will need to talk to the IT. Audio isn't working becasue archive is having a DDoS attack
